packages/components/src/List/Toolbar/ColumnChooserButton/README.md

```diff
@@ -0,0 +1,249 @@
+```javascript
+	{columnChooser &&  <ColumnChooserButton  {...columnChooser}/>}
+```
+
+You have to pass a columnChooser props to the List.
+This object must holds two fields, columns ([ColumnsChooserPropTypes](#columnschooserproptypes))and submit. You can also locked some columns with nbLockedLeftItems.
+```javascript
+import { List } from '@talend/react-components';
+...
+<List {...rest} columnChooser={{
+	columns: [
+		{
+			label: 'first column',
+			order: 1,
+		},
+		{
+			label: 'second column',
+			order: 2,
+		},
+	],
+	nbLockedLeftItems: 1,
+	submit: () => { my function triggered at submit event },
+}} />;
+```
+
+This settings will create a column chooser with two columns. The first one will be locked, because of nbLockedLeftItems: 1.
+
+| Fields | Type | Info
+|---|---|---|
+| columns | [ColumnsChooserPropTypes](#columnschooserproptypes) | populate the popover
+| submit | function | callback trigger when the form is submit
+| nbLockedLeftItems | number | the number of locked items you want, beginning at the left
+
+To keep the columns from the list sync with the inputs of the user, you will need to merge the data from the column chooser with the columns from the list.
+There is a service to help you to do that, [mergedColumnsChooser](#mergedcolumnschooser).
+
+```javascript
+const columnsMergedWithColumnChooser = {
+		...list,
+		columns: columnChooserService.utils.mergedColumnsChooser(list.columns, columnsChooser),
+};
+<List {...rest} columnChooser={...} list={columnsMergedWithColumnChooser} />;
+```
+If you are using the compound list you just have to import the [ColumnChooserButton](#columnchooserButton) and place it where you need it.
+Do not forget to pass an id in this case.
+
+Either way you will have to update the columns props given to the list with the submitted values.
+
+## Components
+### ColumnChooserButton
+
+The button which will trigger the popover with the form and content.
+If you pass children you will override the default column chooser renderer.
+
+| Props | Type | Info
+|---|---|---|
+| children | React.element | the content of the popover
+| columns | array | these columns comes from the list, they will populate the popover
+| id | string | Use as prefix for all the children components
+| submit | function | callback trigger when the form is submit
+| nbLockedLeftItems | number | the number of locked columns you want, beginning at the left
+| t | function | i18n translate
+
+### ColumnChooser
+
+The base of the popover component.
+This component holds the form. If you pass children you will override the default column chooser renderer.
+The [columnChooserContext](#columnchoosercontext) is initialized here.
+
+The component holds reference :
+* Header : [ColumnChooserHeader](#columnchooserheader)
+* Body: [ColumnChooserBody](#columnchooserbody)
+* Footer :[ColumnChooserFooter](#columnchooserfooter)
+
+| Props | Type | Info
+|---|---|---|
+| children | React.element, [React.element] | the content of the popover
```

Comment:
why 2 React.element?

Reply:
I can't put pipe or it will create another column.
I think the comma is ok to list all kind of accepted parameters.